frontend/lib/src/WidgetStateManager.ts

```diff
@@ -586,6 +591,11 @@ export class WidgetStateManager {
   public removeInactive(activeIds: Set<string>): void {
     this.widgetStates.removeInactive(activeIds)
     this.forms.forEach(form => form.widgetStates.removeInactive(activeIds))
+    this.elementStates.forEach((value, key) => {
```

Comment:
to not be to confusing with the `key` from below, it might be better to rename the `key` here to `elementId`